Source/WebKit/UIProcess/API/gtk/DropTargetGtk4.cpp

```diff
@@ -178,7 +176,7 @@ void DropTarget::accept(GdkDrop* drop, std::optional<WebCore::IntPoint> position
         }
 
         m_dataRequestCount++;
-        loadData(supportedMimeTypes[i], [this, transferredFilesFromPortal, mimeType = String::fromUTF8(supportedMimeTypes[i]), cancellable = m_cancellable](GRefPtr<GBytes>&& data) {
+        loadData(mimeType, [this, transferredFilesFromPortal, mimeType = String(mimeType), cancellable = m_cancellable](GRefPtr<GBytes>&& data) {
```

Comment:
I think you shouldn't need the cast to String here because the constructor is implicit. (A rare exception to the rule that single-argument non-explicit constructors are bad!)

Reply:
Good idea, I'll try that before landing.

Reply:
And indeed it worked. Suggestion applied 👌🏼